ifrs17/Import/Importers.ipynb

```diff
@@ -444,28 +467,19 @@
         {
             "cell_type": "code",
             "source": [
-                "public async Task<ImportArgs[]> GetTargetArgsAsync(ImportArgs mainArgs)",
+                "public async Task<ImportArgs[]> GettargetArgsListAsync(ImportArgs mainArgs)",
```

Comment:
It's an strange logic.
You know from the input if mainArgs.Scenario != null, so you don't need any of the logic, as you are returning the input, right?

Reply:
refactored